### ISSUE TITLE
Where do i get the better accurate model for image segmentation?

### ISSUE THREAD

**Issue by charlizesmith** [tensorflow/tensorflow#34702]
Hi,I am using tensor flow lite for Image segmentation within my app.I am using the following gradle dependency for tensor flow lite"    implementation 'org.tensorflow:tensorflow-lite:0.0.0-gpu-experimental'".With this implementation the accuracy of the resultant cutout is not up to the mark .How can i improve the accuracy of image with tensor flow lite?And is there any pre-trained model other than deeplab v3+ for tensor flow.Please Help

**Comment by nutsiepully** [comment]
Hi Charlize,

From your question, it's unclear if the issue is with model accuracy or with the GPU implementation? Can you run your model on CPU and confirm whether you are getting the same accuracy or not.

If it's not the same, we can investigate why the GPU delegate results are different. (Please attach the model in that case.)

DeepLab is currently the latest available image segmentation model.

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ravikyram** [comment]
@charlizesmith 

Any update on this issue please. Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
